src/app/components/views/weather-view/weather-view.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <div class="row mb-5">
+    <div class="col text-center">
+      <div class="card pt-3">
+        <h3>Weather Alerts</h3>
+        <hr>
+        <h5>Active weather alerts in your area:</h5>
+      </div>
+    </div>
+  </div>
+  <ng-container *ngFor="let alert of weatherAlertResponse?.allActiveWeatherAlerts; let i = index">
+    <div class="card mb-2">
+      <div class="row">
+        <div class="col-3 text-center pt-3 block rain">
```

Comment:
will @comm-scripter be submitting his own PR for the cards he was coding?  or are these staying. because i think we discussed having the cards look like the toast for consistency rather than the job cards

Reply:
I'm supposed to be getting with him tonight for this, we've discussed looking over and adding it, but I haven't gotten anything to add yet.